### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by preston-evans98**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,23 +13,61 @@ as of Mar 18, 2023.
...
+At a bare minimum, you should ensure that the verifier rejects...
+
+1. If the order of the blobs in an otherwise valid input is changed
+1. If the sender of any of the blobs is tampered with
+1. If any blob is omitted
+1. If any blob is duplicated
+1. If any extra blobs are added
+
+For compatibility, we also recommend (but don't require) that any logic in the `DaVerifier` be able to build with `no_std`.
+This maximizes your odds of being compatible with
```
Comment (cemozerr) [inline README.md]: I think you forgot to finish this line.

File: README.md [hunk 2]
```diff
@@ -13,23 +13,61 @@ as of Mar 18, 2023.
 Jupiter is a research prototype. It contains known vulnerabilities and should not be used in production under any
 circumstances.
 
-## Getting Started
```
Comment (cemozerr) [inline README.md:16]: Are we sure that we want to remove the getting started instructions?

File: README.md (hunk 1 above)
Comment (preston-evans98) [reply README.md]: Oops, yeah got truncated somewhere haha. Fixing.

File: README.md (hunk 2 above)
Comment (preston-evans98) [reply README.md:16]: Yeah, they really shouldn't have been there in the first place. Jupiter is a library - it just happened to have a `main` function that I used for testing at one point.

Comment (cemozerr) [reply README.md:16]: Makes sense!

File: README.md (hunk 1 above)
Comment (preston-evans98) [reply README.md]: Done!

**Comment by cemozerr** [comment]
LGTM. Seeeend it 🚀

**Review by cemozerr — APPROVED** [review]
LGTM. Send it 🚀

**Pull request merged**